torchinductor/microbench.py

```diff
@@ -22,7 +21,6 @@
 
 def timed(model, example_inputs, times=1):
     synchronize()
-    gc.collect()
```

Comment:
jw/ why did we GC here, and why did we remove it?

Reply:
gc defeats the caching allocator, we want to allow cached allocations.